node/src/main.rs

```diff
@@ -56,6 +56,18 @@ lazy_static! {
         .map(|s| BlockNumber::from_str(&s)
              .unwrap_or_else(|_| panic!("failed to parse env var ETHEREUM_ANCESTOR_COUNT")))
         .unwrap_or(50);
+
+    // Comma separated network names that do not support EIP-1898 (calls by block hash). The only
+    // network known to not support it is fuse, once all known networks support eip 1898 this env
+    // var can be removed.
+    static ref NO_EIP_1898_SUPPORT: Vec<String> = {
```

Comment:
As a rule, I don't want to add more environment variables to configure things; can we make that something that you add to the `features` in `graph-node.toml`? It's a little funky that it's a feature that is one by default, but I would much rather have a feature `no-eip1898` than another environment variable.